mod/wikirate/set/type_plus_right/metric/value_type.rb

```diff
@@ -0,0 +1,54 @@
+
+def all_numeric? metric_values
+  metric_values.each do |_key, values|
+    values.each do |mv|
+      return false unless number?(mv['value'])
+    end
+  end
+  true
```

Comment:
or even shorter and equally fast (but much more confusing because of the negations; I favor the other solution):
`!metric_values.any? {  |_key, values| !values.all? { |mv| number? mv['value'] } }`